index.html

```diff
@@ -1727,12 +1727,14 @@ <h3>Issuer</h3>
         <dl>
           <dt><var id="defn-issuer">issuer</var></dt>
           <dd>
-The value of the <code>issuer</code> <a>property</a> MUST be either a
-<a>URL</a> or an object containing an <code>id</code> <a>property</a>. It is
-RECOMMENDED that the <a>URL</a> in the <code>issuer</code> or its
-<code>id</code> be one which, if dereferenced, results in a document containing
-machine-readable information about the <a>issuer</a> that can be used to
-<a>verify</a> the information expressed in the <a>credential</a>.
+The value of the <code>issuer</code> <a>property</a> MUST either be a
+<a>URL</a>, or be an object containing an <code>id</code> <a>property</a>
+whose value is a <a>URL</a>; in either case, the issuer MUST select this
+<a>URL</a> to identify the issuer in a globally unambiguous
+way. It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Comment:
```suggestion
whose value is a <a>URL</a>; in either case, the issuer selects this
<a>URL</a> to identify itself in a globally unambiguous
way. It is RECOMMENDED that the <a>URL</a> be one which, if dereferenced,
```

Hrm... I can see the testing team complaining that the "MUST" statement here is not testable (and we're trying to avoid non-testable MUST statements). Would a rewording like the above be ok with you @jyasskin? I believe it is still aligned w/ what you wanted to say, but in a way that doesn't create a MUST statement we have to defend against the "... and how are you testing this MUST statement?" conversation w/ the VCWG and implementers.

Reply:
I'm applying this @jyasskin and merging. Please let me know if you disagree, we can do the change you want in a future PR.